go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aiven/terraform-provider-aiven
 go 1.21
 
 require (
-	github.com/aiven/aiven-go-client/v2 v2.5.0
+	github.com/aiven/aiven-go-client/v2 v2.5.1-0.20231228170105-c4df0fe534c0
```

Comment:
please use `v2.6.0`; setting this to a fixed commit will break dependabot's automatic updates